### PR TITLE
E2e

### PR DESCRIPTION
**Pull request by Yu1986**
## Description

Create e2e test for aio-cli-plugin-jwt-auth

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by shazron — CHANGES_REQUESTED** [review]
see https://github.com/adobe/aio-cli-plugin-ims/pull/10#pullrequestreview-354919727

File: e2e/jwt_auth_config.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"client_id": "....",
```
Comment (shazron) [inline e2e/jwt_auth_config.json]: not sure what this file is for

File: e2e/e2e.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+stdout.print = true
+
+test('Generate jwt-auth access token test', async () => {
+  const packagejson = JSON.parse(fs.readFileSync('package.json').toString())
+  const name = `${packagejson.name}`
+  console.log(chalk.blue(`> e2e tests for ${chalk.bold(name)}`))
+
+  console.log(chalk.dim(`    - Generate jwt-auth access token..`))
+  expect(() => { execa.sync('./bin/run', ['jwt-auth:access-token'], { stderr: 'inherit' }) }).not.toThrow()
```
Comment (shazron) [inline e2e/e2e.js:25]: i'm pretty sure this will throw an exception.
run `npm run e2e` . Travis does not run this

**Review by shazron — CHANGES_REQUESTED** [review]

File: e2e/e2e.js (hunk 2 above)
Comment (Yu1986) [reply e2e/e2e.js:25]: it requires ~/.wskprops

ieyu-osx:aio-cli-plugin-jwt-auth jieyu$ cat ~/.wskprops
APIHOST=https://adobeioruntime.net
NAMESPACE=53444_70361
AUTH=xxxxxx

Comment (shazron) [reply e2e/e2e.js:25]: actually if you run it, it will complain about missing `jwt-auth` key.

Comment (Yu1986) [reply e2e/e2e.js:25]: yes, need to fill out the config file: e2e/jwt_auth_config.json

**Review by shazron — APPROVED** [review]

**Pull request merged**